stresstests/test/config.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+const deviceHttp = require('azure-iot-device-http');
+const deviceAmqp = require('azure-iot-device-amqp');
+const deviceMqtt = require('azure-iot-device-mqtt'); 
+
+module.exports = {
```

Comment:
>exports [](http://example.com/codeflow?start=7&length=7)

Curious why you used JS intead of TS.  Is this because of mocha?  Is this still the right test framework to use?

Reply:
Mainly because I wasn't sure how TS interacts with Mocha. Sure, I could have figured it out but I didn't think it was worth the effort.

Reply:
Decided to stick with mocha because it seems to work for our scenario and we can avoid our repo having dependencies on different test runners